lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -129,7 +141,7 @@ class window.Turbolinks
         deleteRefreshNeverNodes(body)
 
       triggerEvent 'page:before-replace'
-      document.documentElement.replaceChild body, document.body
```

Comment:
Yeah. I checked and `document.body.parentNode === document.documentElement`, at least on some pages. I don't know enough about the general dom structure to know if this would always be true though. I could put this a special case if needed